examples/user_guide/1_Plotting_Pitfalls.ipynb

```diff
@@ -113,8 +121,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Points (s=50 alpha=0.1)\n",
-    "blues + reds + reds*blues + blues*reds"
+    "layout = blues + reds + reds * blues + blues * reds\n",
+    "layout.opts(opts.Points(s=50, alpha=0.1))"
```

Comment:
PEP8 says to leave the spacing around * like it was: https://www.python.org/dev/peps/pep-0008

Reply:
"If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies)."

Reply:
To be honest, I think explicit parentheses are often the most readable thing. To maximize readability I would do:

```
layout = blues + reds + (reds * blues) + (blues * reds)
```

But otherwise, without the parentheses I do agree with PEP8 in this case.

Reply:
Ok.  I'd only use parentheses if the items involved are somehow complex, but "reds" and "blues" aren't enough to justify parens, for me...

Reply:
I improved the readability of these expressions in 3de0e14 using parentheses. I prefer this format but if you really don't like it, I'll have to revert to the mixed spacing around the operators.

Reply:
I don't prefer it but don't mind; just leave it as you have it now.